scripts/ci/collect-infrastructure-logs.sh

```diff
@@ -34,7 +34,10 @@ proxy_pid=$!
 
 sleep 5 # Let kubectl proxy stabilize
 mkdir -p "${log_dir}"/infrastructure
-curl -s http://localhost:8001/logs/kube-apiserver.log > "${log_dir}"/infrastructure/kube-apiserver.log
+curl -v --retry 8 --retry-connrefused --retry-max-time 120 --continue-at - \
```

Comment:
alternatively we could use 
https://github.com/stackrox/stackrox/blob/be0d2377d628f3efa3e2600d7a9ad4a963eeb916/scripts/lib.sh#L158